### DISCUSSION TITLE
npx hardhat node throwing error

### DISCUSSION THREAD

**Discussion by Kriptocourier** [NomicFoundation/hardhat#3737] [Troubleshooting]
![hh2](https://user-images.githubusercontent.com/125918199/222857159-55b83de5-c6c5-41f6-8b80-8d975f3e2e7b.png)

The above screenshot shows the error code that is thrown when i try to run the hardhat node.

Any ideas?

Be kind, I'm suuuuper new to coding

**Comment by fvictorio** [comment]
You are probably using the word `tasks` instead of `task` in your config.
